ci-operator/step-registry/ipi/deprovision/openstack/collectartifacts/ipi-deprovision-openstack-collectartifacts-commands.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cp ${SHARED_DIR}/* ${ARTIFACT_DIR}
```

Comment:
I think this will leak install-config.yaml, exposing a pull secret at least. It should be more targetted to openstack-specific artifacts

Reply:
cleaned up so we specifically list files we want to collect. excluding install-config.yaml at this time.

Reply:
done